src/C/cpl_vsi.jl

```diff
@@ -1,59 +1,51 @@
-# Julia wrapper for header: /usr/local/include/cpl_vsi.h
-# Automatically generated using Clang.jl wrap_c, version 0.0.0
-
-
+# Julia wrapper for header: cpl_vsi.h
+# Automatically generated using Clang.jl
 
 """
     VSIFOpen(const char * pszFilename,
              const char * pszAccess) -> FILE *
 """
-function VSIFOpen(arg1, arg2)
-    ccall((:VSIFOpen, libgdal), Ptr{FILE}, (Cstring, Cstring), arg1, arg2)
+function VSIFOpen()
+    ccall((:VSIFOpen, libgdal), Ptr{Cint}, ())
```

Comment:
what happened to the arguments here?

Reply:
Don't know, it doesn't appear to have changed [in the docs](https://gdal.org/api/cpl.html#cpl__vsi_8h_1ae3cfa1605ce32e78fddb534bba7742f5). We should look into that, though that means digging into Clang.

Reply:
Wait the docs I linked to are VSIOpen**L**, not VSIOpen. So I guess that just changed in the API?

Reply:
linking here for now: https://lists.osgeo.org/pipermail/gdal-dev/2006-October/010478.html